src/api/spec/models/branch_package_spec.rb

```diff
@@ -97,5 +97,14 @@
         end
       end
     end
+
+    context 'Newly created Package' do
+      it 'retains url' do
+        leap_project
+        branch_apache_package.branch
+        project = Package.where(name: 'apache2').where.not(id: apache.id).first
```

Comment:
that `project` variable is actually a package. But that opens a side discussion: why not copy the url in the branch project also? It's just one bug report away

Reply:
made changes as reqiured @coolo. One thing worth looking at - https://github.com/openSUSE/open-build-service/pull/11352/files#r666173752